data/sql_setup/prepare_schedule_a.sql

```diff
@@ -26,24 +26,31 @@ begin
         two_year_transaction_period_new = get_transaction_year(new.contb_receipt_dt, new.rpt_yr);
 
         if two_year_transaction_period_new >= start_year then
+            delete from ofec_sched_a_queue_new where sub_id = new.sub_id;
             insert into ofec_sched_a_queue_new values (new.*, timestamp, two_year_transaction_period_new);
         end if;
+
         return new;
     elsif tg_op = 'UPDATE' then
         two_year_transaction_period_new = get_transaction_year(new.contb_receipt_dt, new.rpt_yr);
         two_year_transaction_period_old = get_transaction_year(old.contb_receipt_dt, old.rpt_yr);
 
         if two_year_transaction_period_new >= start_year then
+            delete from ofec_sched_a_queue_new where sub_id = new.sub_id;
```

Comment:
I have the same understanding.  @LindsayYoung when you mentioned doing a sort and grabbing the latest, what column did you have in mind to sort on?  It would have to be a column like contb_receipt_dt right?